code/obj/item/bowling.dm

```diff
@@ -25,7 +38,7 @@
 	proc/hitHard(var/mob/hitMob, var/mob/user)
 		hitMob.visible_message(SPAN_ALERT("[hitMob] is knocked over by [user]'s [src]!"))
 
-		src.damage(hitMob, 10, 15, user)
+		src.damage(hitMob, 20, 30, user)
```

Comment:
```suggestion
		src.damage(hitMob, 15, 20, user)
```
30 is a lot of damage and #19525 made this able to combo with vending machines/glass tables etc.